### ISSUE TITLE
Mysql/MariaDB UTF8 defaults

### ISSUE THREAD

**Issue by dereuromark** [cakephp/app#1010]
This is a (multiple allowed):

* [] bug
* [x] enhancement

### What you did
See https://github.com/ddev/ddev/issues/6428

### What you expected to happen
I would imagine either the ddev/devenv to auto install this properly, or as another stategy outlined in the related ticket maybe the CakePHP installer could auto adjust this also internally
https://github.com/cakephp/app/blob/ca45a282941596d4ddadd9ca94bf39fde2951798/src/Console/Installer.php#L70

**Comment by dereuromark** [comment]
So https://github.com/cakephp/app/compare/installer-mysql would auto adjust this for every install
Any idea how to get the DB connection here dynamically and only switch then?

**Comment by dereuromark** [comment]
I tried
```php
        try {
            Configure::config('default', new PhpConfig());
            Configure::load('app', 'default', false);
        } catch (\Exception $e) {
            return;
        }
```

but that's too early without other bootstrapping
````
  [Error]                                                 
  Undefined constant "Cake\Core\Configure\Engine\CONFIG"  
```

**Comment by markstory** [comment]
Is this solved by #1011?
